utils.py

```diff
@@ -119,6 +130,9 @@ def email_stash_load_request(logger, email, req, minutes_expiry):
     msg = f"You have a pending stash load request waiting!<br/><br/>Confirm the request came from you <a href='{url}'>here</a><br/><br/>Confirm within {minutes_expiry} minutes"
     send_email(logger, "Confirm your stash request", msg, email)
 
+def email_notification_alert(logger, subject, msg, recipient):
```

Comment:
sends the email notification when the balance is getting low or the claimable is higher than the amount in ops wallet.